### DISCUSSION TITLE
Click to move the tab to a new window, can the window screen be as large as the controlled screen? (+1)

### DISCUSSION THREAD

**Discussion by youhers** [rustdesk/rustdesk#5572] [Feature Request]
After connecting, you need to click to adjust the window to make the window as large as the picture, if you automatically adapt the frame size of the work according to the size of the screen after connecting It would be great, for now, after moving out of one screen from the two controlled dialogs The picture is very small You need to click to adjust the window This is not so convenient, I bought the professional version, looking forward to subsequent modifications, hard work

连接后需要点击调节窗口才能让窗口与画面一样大，如果连接后自动根据画面大小去适配工作的边框大小 那就太好了，就目前而言从两个被控对话框移出来一个画面后  画面很小 需要点击调节窗口 这点并不是那么方便，我以购买专业版，期待后续的修改，辛苦了

**Comment by rustdesk** [comment]
Thanks, all feature request from pro users we will listen to, and put in priority.

**Comment by youhers** [comment]
Good work you guys RustDesk from the year before last to this year has been very good to use
辛苦你们了 rustdesk从前年到今年  已经很好用了
